p2p/module_raintree_test.go

```diff
@@ -15,12 +15,14 @@ import (
 	"testing"
 	"time"
 
+	"github.com/pokt-network/pocket/runtime"
 	"github.com/pokt-network/pocket/shared/debug"
 	"github.com/pokt-network/pocket/shared/test_artifacts"
 
 	"github.com/golang/mock/gomock"
 	typesP2P "github.com/pokt-network/pocket/p2p/types"
 	mocksP2P "github.com/pokt-network/pocket/p2p/types/mocks"
+	typesPers "github.com/pokt-network/pocket/persistence/types"
```

Comment:
This is a step backwards when it comes to encapsulation. You've introduced cross module imports. Please revert / fix to use `test_artifacts`

Reply:
Not happening anymore 

https://github.com/pokt-network/pocket/pull/235/files#diff-a8351346d15469e6fb8938609c72b7b4387e5d77fd634fc64ed82327ad40fb5dR4-R25